backend/src/data/seed-data/vendors-seed.ts

```diff
@@ -0,0 +1,12 @@
+const vendorsSeed = [
+  {
+    name: 'Udemy',
+    key: 'Key1',
+  },
+  {
+    name: 'Coursera',
+    key: 'Key2',
```

Comment:
for future, on such key-name pairs in tables -> key should be a snake_cased value from name